packages/@lwc/integration-karma/scripts/shared/options.js

```diff
@@ -23,6 +23,7 @@ const DISABLE_SYNTHETIC_SHADOW_SUPPORT_IN_COMPILER = Boolean(
     process.env.DISABLE_SYNTHETIC_SHADOW_SUPPORT_IN_COMPILER
 );
 const NODE_ENV_FOR_TEST = process.env.NODE_ENV_FOR_TEST;
+const API_VERSION = process.env.API_VERSION && parseInt(process.env.API_VERSION, 10);
```

Comment:
True, but it is an old habit of mine to do this, since older JS versions [potentially use 8](https://stackoverflow.com/a/6611854) as the radix if the string starts with a `0`. So I like being explicit. :)